docs_user/modules/proc_deploying-backend-services.adoc

```diff
@@ -253,18 +241,18 @@ spec:
     templates:
       openstack:
         secret: osp-secret
-        replicas: 1
```

Comment:
Just FYI: I updated the replicas as described in https://issues.redhat.com/browse/OSPRH-5645?focusedId=25012945&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-25012945

Reply:
+1 that's a good catch, we could perhaps delete the replicas key instead though and rely on the default which should be 3. The `replicas: 1` here came from us deploying slimmed-down environments for development.